### ISSUE TITLE
Portal app: narrowing down a selection by removing items in a filter not possible

### ISSUE THREAD

**Issue by atverm** [ICOS-Carbon-Portal/data#244]
For example when selecting all atmosphere data L1+L2 starting at state https://data.icos-cp.eu/portal/#%7B%22filterCategories%22%3A%7B%22level%22%3A%5B1%2C2%5D%2C%22theme%22%3A%5B%22atmosphere%22%5D%7D%2C%22tabs%22%3A%7B%22resultTab%22%3A1%7D%7D it is not possible to remove the meteorological data by removing data types ICOS ATC Meteo release etc from the Data Type filter. One solution would be to add a cross to each element in the filter that one can click to exclude this item from the list. Otherwise the user has to clear all filters and then first select all the wanted datatypes before further building the filter

**Comment by jonathanthiry** [comment]
Which data types are you interested in? Isn't it possible to directly select them?

**Comment by atverm** [comment]
If I am interested in 24 data types out of 26 it is more convenient to deselect 2 then to go back and have to select 24. Is in general so also with stations for example, where I would have to pick 80 one by one out of 81, and yes I could use the map there, if I know where the station is.

Alex

**Comment by mirzov** [comment]
I think an adequate solution to this could be adding an "Add all" functionality to the multiselect dropdowns. After adding all, removing the undesired few becomes easy.

**Comment by roger-groth** [comment]
An alternative solution: Add ability to invert selection. If you activate "inverted mode" for filter Theme and select "Ocean data", all themes EXCEPT "Ocean data" is added. Inverted mode can also be activated on filters with existing values giving the end user an easy way to switch between "show only this" and "show anything but this".

Pro - less clicking than add all and then find the ones you want to exclude
Con - Less intuitive

**Comment by mirzov** [comment]
Maybe if we just limit ourselves to an "invert selection" button, that will be enough? Then we have both less clicking, and still intuitive behavior -- no extra "mode", just a helpful operation.

**Comment by atverm** [comment]
No, that is not what was meant. From the current selection only a few have to be removed. So what would help is that all elements would have the cross on the label to remove it, even if all are there as the default. Now labels only get a cross when you added them to the selection. When none are selected, all are selected, without that you can deselect them any more through the cross on the label.
